herbert_core/classes/instrument_classes/instrument/detectors/@IX_det_He3tube/covariance.m

```diff
@@ -1,48 +1,60 @@
-function val = covariance (obj, npath_in, varargin)
+function val = covariance (obj, varargin)
 % Covariance of points of absorption in a 3He cylindrical tube
 %
 %   >> val = covariance (obj, npath, wvec)
-%   >> val = covariance (obj, npath, ind, wvec)
+%   >> val = covariance (obj, ind, npath, wvec)
```

Comment:
The only places that this method has been used are within Horace; it is very unlikely that any users have called it (and similar methods for detector mean position etc. where the same change has been made). Although not the nicest practice, having ind first is what is expected by the fast sampling methods rand_ind and func_eval_ind of object_lookup. Ind is the natural first argument ("I want the covariance of these detectors under these conditions - and by the way if I don't tell you, make it all the detectors"). Rather than shift the order of arguments, I'd rather demand that if you want all of them, you make the call >> val = covariance (obj, 1:obj.ndet, npath, wvec), but that is rather silly if it just a single detector...